### PR TITLE
Use device_id to identify cases imported from OpenMRS

### PR DESCRIPTION
**Pull request by kaapstorm**
This change should make it easy to see that the cases were imported, and not created by a user.

Feature flag: 'Enable OpenMRS integration'

### CODE REVIEW

File: corehq/motech/openmrs/const.py [hunk 1]
```diff
@@ -21,6 +21,10 @@
     (IMPORT_FREQUENCY_MONTHLY, _('Monthly')),
 )
 
+# device_id for cases added/updated by an OpenmrsImporter.
+# OpenmrsImporter ID is appended to this.
+OPENMRS_IMPORTER_DEVICE_ID = 'openmrs-importer-'
```
Comment (czue) [inline corehq/motech/openmrs/const.py]: nit: could change to `OPENMRS_IMPORTER_DEVICE_ID_PREFIX` just to indicate that it's not a complete ID

File: corehq/motech/openmrs/tasks.py [hunk 2]
```diff
@@ -144,7 +149,7 @@ def import_patients_of_owner(requests, importer, domain_name, owner, location=No
     submit_case_blocks(
         [cb.case.as_string() for cb in case_blocks],
         domain_name,
-        username=owner.username,
+        device_id=OPENMRS_IMPORTER_DEVICE_ID + importer.get_id,
```
Comment (czue) [inline corehq/motech/openmrs/tasks.py]: nit: I think `'{}{}'.format(OPENMRS_IMPORTER_DEVICE_ID, importer.get_id)` would be a slightly more safe way to do this (though doesn't matter if both are always strings).

**Review by czue — APPROVED** [review]
feel free to ignore my comments

File: corehq/motech/openmrs/tasks.py (hunk 2 above)
Comment (kaapstorm) [reply corehq/motech/openmrs/tasks.py]: :+1: It's a good principle regardless.

**Pull request merged**